sources/osgShadow/ShadowTexture.js

```diff
@@ -77,8 +77,9 @@ define( [
 
             }.bind( this ) );
 
-
+            // tells shader where to find it
```

Comment:
I dont understand the extra logic, could you explains why it's not like Texture attribute ?